kitchen.ec2.yml

```diff
@@ -27,54 +14,15 @@ driver:
   require_chef_for_busser: false
   privileged: true
 
-  # image_id: ami-06e9e5a68a55a62da
-
-provisioner:
-  name: puppet_apply
-  sudo: true
-  require_chef_for_busser: false
-  manifests_path: ./spec/puppet/manifests
-  modules_path: ./spec/puppet/modules
-  hiera_config_path: ./spec/puppet/hiera.yaml
-  hiera_data_path: ./spec/puppet/hiera
-  resolve_with_librarian_puppet: true
-  ignore_spec_fixtures: true
-  puppet_version: "5.5.19"
+lifecycle:
+  pre_converge:
+    - remote: |
+        echo "NOTICE - Installing needed packages"
+        sudo yum install -y bc bind-utils redhat-lsb-core vim
+        echo "NOTICE - Updating the ec2-user to keep sudo working"
+        sudo chage -d $(( $( date +%s ) / 86400 )) ec2-user
+        echo "NOTICE - updating ec2-user sudo config"
+        sudo chmod 600 /etc/sudoers && sudo sed -i'' "/ec2-user/d" /etc/sudoers && sudo chmod 400 /etc/sudoers
 
 transport:
   name: ssh
-  # username: ec2-user
-  ssh_key: ~/.ssh/<%= ENV[('AWS_SSH_KEY_ID')] %>.pem
-
-verifier:
-  name: inspec
-  sudo: true
-  inspec_tests:
-    - name: disa_stig-el7
-      path: .
-  input_files:
-    - inputs.example.yml
-  reporter:
-    - cli
-    - json:test-result.json
-
-platforms:
-  - name: rhel-7
-
-suites:
-  - name: default
-    lifecycle:
-      post_create:
-        - remote: echo "NOTICE - Installing needed packages"
-        - remote: sudo yum install -y bc bind-utils redhat-lsb-core vim
-        - remote: echo "NOTICE - Updating the ec2-user to keep sudo working"
-        - remote: sudo chage -d $(( $( date +%s ) / 86400 )) ec2-user
-        - remote: echo "NOTICE - updating ec2-user sudo config"
-        - remote: sudo chmod 600 /etc/sudoers && sudo sed -i'' "/ec2-user/d" /etc/sudoers && sudo chmod 400 /etc/sudoers
-
-    provisioner:
-      manifest: stig.pp
-    verifier:
-      name: inspec
-      inputs:
-        disable_slow_controls: true
```

Comment:
We need to check the `default` inputs for the profile to ensure that we are in fact disabling the long-running controls. I think we have a PR asking that this default be set back to false so in our CICD we will want to set it to `true`